extensions/expression-language/Expression.cpp

```diff
@@ -25,6 +25,10 @@
 #include <functional>
 #include <string>
 
+#ifdef GetObject
+#undef GetObject
+#endif
```

Comment:
As in #1129, I would change all occurrences of this to:
```suggestion
#undef GetObject  // windows.h #defines GetObject = GetObjectA or GetObjectW, which conflicts with rapidjson
```

(not sure if they are the same lines; if so then I guess resolving merge conflicts will take care of this)

Reply:
There are more lines here. I started getting these errors all the time for some reason. It seemed that even standard headers somehow got this defined. Not sure what triggered the explosion of these definitions, but I added one undef near every rapidjson include.
I'll try removing them to see if they occur in C++17 mode or are specific to /std:c++latest.

Reply:
They are gone without issues, I'll reintroduce the necessary ones in the C++20 followup.